packages/boilerplates/ScriptFailedError.ts

```diff
@@ -0,0 +1,14 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Dolittle. All rights reserved.
+*  Licensed under the MIT License. See LICENSE in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+export class ScriptFailedError extends Error {
+    constructor(...args: any[]) {
+        super(...args);
+        Error.captureStackTrace(this, ScriptFailedError);
+    }
+
+    static get new() {
```

Comment:
Use the Exception: https://github.com/dolittle-fundamentals/JavaScript.Fundamentals/tree/master/Source/exceptions 

then it becomes much clearer with `[ExceptionName].throw()`